### ISSUE TITLE
yuba instream docs

### ISSUE THREAD

**Issue by thesadie** [FlowWest/cvpiaHabitat#19]
finish markdown edits

**Comment by ErinCain** [comment]
I cannot find the data source.  @mtompkins1101 if you could point me in the right direction that would be helpful.

**Comment by ErinCain** [comment]
Found two potential data sources but not sure that either fits properly. I am waiting for Mark to help confirm which source the data came from before continuing to work on this one.

**Comment by ErinCain** [comment]
@thesadie have we replaced the Mark Gard IFIMWUA data with the VA data provided by the district in 2019? If so can you provide that reference.

**Comment by ErinCain** [comment]
It looks like the data is from the IFIMWUA spreadsheet but I don't know where that is coming from and Mark thought that it may have been the more recent data.

**Comment by ErinCain** [comment]
Also need to check with @mtompkins1101 to figure out what to do with flows that are under 400 because these don't seem consistent.

**Comment by ErinCain** [comment]
Waiting on the VA source inputs before future edits. I sent a pull request for what I had already done that is mostly just formatting.
